cpp/include/cudf/aggregation.hpp

```diff
@@ -180,6 +181,11 @@ std::unique_ptr<Base> make_m2_aggregation();
 /**
  * @brief Factory to create a VARIANCE aggregation
  *
+ * In rolling window aggregation, when there is only one valid observation
+ * in the window, the result is null.
+ *
+ * Chrono type is unsupported in rolling window VARIANCE aggregation.
```

Comment:
What happens? Is an exception thrown for chrono types?

Reply:
Yes, an "Invalid aggregation type/pair" exception will be thrown.

Reply:
Use `@throw` do document that here then. And the one below.